netlify.toml

```diff
@@ -10,7 +10,7 @@
 [[headers]]
   for = "/*"
   [headers.values]
-    X-Frame-Options = "DENY"
+    X-Frame-Options = "sameorigin"
```

Comment:
copy-paste from chat just in case:

> from what i can see netlify.toml doesn't change headers in production actually 🤔 i forgot, what webhosting we're using currently? cloudflare proxies requests to Netlify, Fleek or something else? 

> we may need to edit headers in some other place (and there are some other security headers to add, but better to add them in separate task just in case)